x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_form.tsx

```diff
@@ -758,6 +791,25 @@ export const AlertForm = ({
           <EuiHorizontalRule />
           <EuiFormRow
             fullWidth
+            labelAppend={
+              hasDisabledByLicenseAlertTypes && (
+                <EuiTitle size="xxs">
+                  <h5>
```

Comment:
Not sure how to get all the way to this screen in the UI but an `h5` strikes me as an odd choice. Do we already have an `h1` - `h4` on this page?

Reply:
Removed